assets/responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -9,8 +9,6 @@ dependencies:
   - pip:
     - responsibleai~=0.30.0
     - raiwidgets~=0.30.0
-    - pyarrow
-    - pyarrow-hotfix>=0.5
```

Comment:
We don't need pyarrow-hotfix?

Reply:
no, the only reason we need it is if we are using pyarrow with the vulnerability https://github.com/advisories/GHSA-5wvp-7f3h-6wmm: 
<img width="395" alt="image" src="https://github.com/Azure/azureml-assets/assets/42190563/f82834c3-d635-4292-8474-830e1b0440b6">

Reply:
Sounds good.